frontend/device/sdl/device.lua

```diff
@@ -2,6 +2,14 @@ local Event = require("ui/event")
 local Generic = require("device/generic/device")
 local SDL = require("ffi/SDL2_0")
 local logger = require("logger")
+local ffi = require("ffi")
```

Comment:
Not quite sure where it should go really due to what it is, but I'd say either at the very top or above logger, definitely not there. 😁